### ISSUE TITLE
question about assertion in model.py

### ISSUE THREAD

**Issue by bowenyou** [SymposiumOrganization/NeuralSymbolicRegressionThatScales#7]
Hi guys, sorry for asking so many questions lately XD

Just wondering about this assertion in line 88 of model.py:
```
assert not torch.isnan(enc_src).any()
```
It seems when I am running the training script, this assertion fails consistently which halts training. Have you guys run into a similar issue before?

**Comment by TommasoBendinelli** [comment]
Hi, I would check where are these nans coming from (input to the network or the encoder processing). In the first case you would need to get rid of them or handle them in the float2bit function. In the latter case I would just try to reduce the learning rate.
